examples/lab_devices/SignatoneProber.yaml

```diff
@@ -0,0 +1,13 @@
+transfer_layer:
```

Comment:
maybe you also want to add a (commented) example to the `probestation.py` file to make the example more obvious?

Reply:
Any suggestions of which comments/examples might be missing?

Reply:
you only added the `yaml` file, which dowsn't work as an example on it's own. you could add another (commented out) instantiation to the actual callable example in `probestation.py`, as is done for most other lab devices. it's not that imrportant, though.